OWNERS

```diff
@@ -1,4 +1,8 @@
 approvers:
   - bobgy
+  - james-jwu
   - jlewi
-  - rmgogogo
\ No newline at end of file
+  - karkumar
```

Comment:
```suggestion
```

Per #339 @karkumar wants to use @kramachandran that said lets remove @paveldournov and @karkumar as they likely won't actually be reviewing PRs to the google groups.